lms/templates/instructor/instructor_dashboard_2/cohorts.underscore

```diff
@@ -47,5 +47,12 @@
                 ) %>
             </p>
         </div>
+
+        <hr class="divider divider-lv1" />
+
+        <!-- Discussion Topics. -->
+        <a class="toggle-cohort-management-discussions" href="#cohort-discussions-management"><%- gettext('Assign cohorts to discussion topics') %></a>
```

Comment:
addressed in previous comment. This text should be changed to ""Specify whether discussion topics are divided by cohort".